### PR TITLE
Support mmcli

### PR DESCRIPTION
**Pull request by cherryjm**
- add MANIFEST.in
- add metafile for 2 topdown models (resnet, hrnet)
- add model_zoo.yml
- add mminstall.txt
- modify setup.py

### CODE REVIEW

File: configs/top_down/resnet/metafile.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+  - Name: res152_mpii_256x256
+    In Collection: PoseResNet
+    Config: configs/top_down/resnet/mpii/res152_mpii_256x256.py
+    Metadata:
+      Training Data: MPII
+      FLOPs: 17020000000
+      Parameters: 68640000
+    Results:
+      - Task: 3D Human Pose Estimation
```
[the remark this answers is not in the document]
Comment (cherryjm) [reply configs/top_down/resnet/metafile.yml:176]: Sorry... will fix it

**Review by jin-s13 — APPROVED** [review]

**Comment by jin-s13** [comment]
CI fails.

**Comment by cherryjm** [comment]
> CI fails.

That's weird😢I haven't modified any python files except setup.py.

**Comment by ly015** [comment]
The CI problem is caused by incompatibility with opencv-python 4.5.2. It will be fixed by #635. Please rebase to master after that PR has been merged.

**Review by ly015 — APPROVED** [review]

**Pull request merged**